wolfssl/wolfcrypt/settings.h

```diff
@@ -2766,8 +2766,9 @@ extern void uITRON4_free(void *p) ;
     #define NO_SESSION_CACHE_REF
 #endif
 
-/* DTLS v1.3 requires 64-bit number wrappers */
-#if defined(WOLFSSL_DTLS13) && !defined(WOLFSSL_W64_WRAPPER)
+/* TLS v1.3 requires 64-bit number wrappers */
+#if defined(WOLFSSL_TLS13) || defined(WOLFSSL_DTLS13)
```

Comment:
Are w64wrapped used as well for dtls1.2 only code?

Reply:
Necessary when `WOLFSSL_DTLS_DROP_STATS` is defined.